### PR TITLE
Add wiki for the Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by shricodev**
A wiki page for Hacktoberfest 2023 has been added, and the sidebar has been updated to reflect the change.

### CODE REVIEW

File: Hacktoberfest-2023.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+4. Fix the issue by creating some changes and sending a pull request for review, following our [PR instructions](https://github.com/oppia/oppia/wiki/Make-a-pull-request).
+5. Go through the code review process until you receive approval and the PR gets merged.
+6. To earn the digital reward kit: repeat until you've submitted at least 4 pull requests!
+
+## For non-code contributors
+
+If you’re not a coder, don’t worry! Hacktoberfest has introduced a [low/non-code contribution option](https://hacktoberfest.com/about/#low-or-non-code), so you can still participate, and there are lots of opportunities to do so :)
+
+We have a lot of non-code-contribution opportunities available, including giving talks, organizing case studies, translating, illustrations, graphic design and creating practice questions. To learn more about these opportunities, please check out our [hacktoberfest-2023-non-code-contributions GitHub repo](https://github.com/oppia/hacktoberfest-2023-non-code-contributions) and its [issue tracker](https://github.com/oppia/hacktoberfest-2023-non-code-contributions/issues)!
```
Comment (shricodev) [inline Hacktoberfest-2023.md]: Hey @seanlip, it looks like we don't have that repository set up just yet.

**Review by Ash-2k3 — APPROVED** [review]
Thanks @shricodev, LGTM, ping me to merge once you get reply from @seanlip re: your comment.

File: Hacktoberfest-2023.md (hunk 1 above)
Comment (seanlip) [reply Hacktoberfest-2023.md]: I'd say drop the whole non-code-contribution opportunities thing for now. I don't think that really got much takeup last year and it was a bit hard to administer.

Comment (shricodev) [reply Hacktoberfest-2023.md]: Done!

**Comment by shricodev** [comment]
@seanlip @Ash-2k3 If this looks good. let's merge this quickly as I have already provided the link to the wiki to Yiga.

**Comment by seanlip** [comment]
done, thanks!

**Pull request merged**